dash/testing/application_runners.py

```diff
@@ -222,8 +222,7 @@ def stop(self):
                 logger.info("proc.terminate with pid %s", self.proc.pid)
                 self.proc.terminate()
                 if self.tmp_app_path and os.path.exists(self.tmp_app_path):
-                    logger.debug("removing temporary app path %s",
-                                 self.tmp_app_path)
+                    logger.debug("removing temporary app path %s", self.tmp_app_path)
```

Comment:
Surprised these bits of formatting aren't caught by our CI linting job - does this PR fix that? Looks like you're making largely the same changes @dannysepler made in #1277

Reply:
Yeah, I'm also a bit confused at this. The husky commit hook should be running essentially the same thing as CI. Maybe there are still some dependencies whose versions aren't fixed that would account for the different behavior?